codewof/programming/content/en/speeders/test-case-5-code.txt

```diff
@@ -0,0 +1 @@
+print(speeding_tickets([-103, -104, -105, -106], -100))
```

Comment:
This is an odd test case, since it makes no sense in the context, so should be removed.